### ISSUE TITLE
[hunspell] build failure

### ISSUE THREAD

**Issue by vadi2** [microsoft/vcpkg#10565]
**Host Environment**
- OS: macOS
- Compiler: 

-- The C compiler identification is AppleClang 11.0.0.11000033
-- The CXX compiler identification is GNU 9.3.0

**To Reproduce**
`./vcpkg install hunspell`

**Failure logs** 
```
2020-03-26T10:46:35.7141060Z [1/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/unmunch.dir/src/tools/unmunch.cxx.o -MF CMakeFiles/unmunch.dir/src/tools/unmunch.cxx.o.d -o CMakeFiles/unmunch.dir/src/tools/unmunch.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/unmunch.cxx
2020-03-26T10:46:35.7144180Z [2/40] : && /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/unmunch.dir/src/tools/unmunch.cxx.o  -o unmunch   && :
2020-03-26T10:46:35.7149780Z [3/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/chmorph.dir/src/tools/chmorph.cxx.o -MF CMakeFiles/chmorph.dir/src/tools/chmorph.cxx.o.d -o CMakeFiles/chmorph.dir/src/tools/chmorph.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/chmorph.cxx
2020-03-26T10:46:35.7151350Z [4/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/hunzip.dir/src/tools/hunzip.cxx.o -MF CMakeFiles/hunzip.dir/src/tools/hunzip.cxx.o.d -o CMakeFiles/hunzip.dir/src/tools/hunzip.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunzip.cxx
2020-03-26T10:46:35.7153290Z [5/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/firstparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/firstparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/firstparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/firstparser.cxx
2020-03-26T10:46:35.7154610Z [6/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -I. -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o -MF CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o.d -o CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx
2020-03-26T10:46:35.7155280Z FAILED: CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o 
2020-03-26T10:46:35.7156960Z /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -I. -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o -MF CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o.d -o CMakeFiles/hunspell.dir/src/tools/hunspell.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx
2020-03-26T10:46:35.7157900Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:1311:24: error: no matching function for call to 'scanline'
2020-03-26T10:46:35.7158100Z           char* temp = readline(gettext("Replace with: "));
2020-03-26T10:46:35.7158250Z                        ^~~~~~~~
2020-03-26T10:46:35.7158840Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:181:18: note: expanded from macro 'readline'
2020-03-26T10:46:35.7159030Z #define readline scanline
2020-03-26T10:46:35.7159160Z                  ^~~~~~~~
2020-03-26T10:46:35.7159780Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:582:7: note: candidate function not viable: 1st argument ('const char *') would lose const qualifier
2020-03-26T10:46:35.7160010Z char* scanline(char* message) {
2020-03-26T10:46:35.7160160Z       ^
2020-03-26T10:46:35.7160780Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:1395:24: error: no matching function for call to 'scanline'
2020-03-26T10:46:35.7160970Z           char* temp = readline(gettext("New word (stem): "));
2020-03-26T10:46:35.7161110Z                        ^~~~~~~~
2020-03-26T10:46:35.7161700Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:181:18: note: expanded from macro 'readline'
2020-03-26T10:46:35.7162170Z #define readline scanline
2020-03-26T10:46:35.7162300Z                  ^~~~~~~~
2020-03-26T10:46:35.7162970Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:582:7: note: candidate function not viable: 1st argument ('const char *') would lose const qualifier
2020-03-26T10:46:35.7163170Z char* scanline(char* message) {
2020-03-26T10:46:35.7163310Z       ^
2020-03-26T10:46:35.7163900Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:1423:18: error: no matching function for call to 'scanline'
2020-03-26T10:46:35.7164100Z           temp = readline(gettext("Model word (a similar dictionary word): "));
2020-03-26T10:46:35.7164240Z                  ^~~~~~~~
2020-03-26T10:46:35.7164820Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:181:18: note: expanded from macro 'readline'
2020-03-26T10:46:35.7165050Z #define readline scanline
2020-03-26T10:46:35.7165140Z                  ^~~~~~~~
2020-03-26T10:46:35.7165770Z /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/tools/hunspell.cxx:582:7: note: candidate function not viable: 1st argument ('const char *') would lose const qualifier
2020-03-26T10:46:35.7166160Z char* scanline(char* message) {
2020-03-26T10:46:35.7166690Z       ^
2020-03-26T10:46:35.7166930Z 3 errors generated.
2020-03-26T10:46:35.7168210Z [7/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/xmlparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/xmlparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/xmlparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/xmlparser.cxx
2020-03-26T10:46:35.7169610Z [8/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/testparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/testparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/testparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/testparser.cxx
2020-03-26T10:46:35.7171880Z [9/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/manparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/manparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/manparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/manparser.cxx
2020-03-26T10:46:35.7174500Z [10/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/latexparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/latexparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/latexparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/latexparser.cxx
2020-03-26T10:46:35.7176760Z [11/40] /Applications/Xcode_11.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -DHUNSPELL_STATIC -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/hunspell -I/Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers -fPIC -O3 -DNDEBUG -isysroot /Applications/Xcode_11.3.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -MD -MT CMakeFiles/testparser.dir/src/parsers/textparser.cxx.o -MF CMakeFiles/testparser.dir/src/parsers/textparser.cxx.o.d -o CMakeFiles/testparser.dir/src/parsers/textparser.cxx.o -c /Users/runner/runners/2.165.2/work/Mudlet/Mudlet/3rdparty/vcpkg/buildtrees/hunspell/src/v1.7.0-658f8373e7/src/parsers/textparser.cxx
2020-03-26T10:46:35.7177440Z ninja: build stopped: subcommand failed.
```

**Additional context**
Running this using https://github.com/vadi2/Mudlet/runs/536127951?check_suite_focus=true

**Comment by JackBoosY** [comment]
Confirmed.

**Comment by JackBoosY** [comment]
Hi @vadi2, thanks for reporting this issue!
I've fixed it in #10574, please update vcpkg and rebuild hunspell.

Thanks.

**Comment by vadi2** [comment]
I can confirm that it builds now. Thanks!
